test/y2storage/boot_requirements_checker_x86_test.rb

```diff
@@ -290,14 +290,31 @@
       end
 
       context "when proposing a new EFI partition" do
+
         let(:efi_part) { find_vol("/boot/efi", checker.needed_partitions(target)) }
         let(:desired_efi_part) { find_vol("/boot/efi", checker.needed_partitions(:desired)) }
         # Default values to ensure proposal of EFI partition
         let(:efiboot) { true }
         let(:efi_partitions) { [] }
 
         include_examples "proposed EFI partition basics"
-        include_examples "flexible size EFI partition"
+
+        context "and BLS bootloader as default" do
+          before do
+            allow(Yast::Linuxrc).to receive(:InstallInf).with("NO_BLS_BOOT").and_return(0)
+          end
+
+          include_examples "EFI partition for BLS bootloaders"
+        end
+
+        context "and no BLS bootloader as default" do
```

Comment:
Similar to the previous one. Of course, this context needs to be here (since it explain a non-default behavior) but I would phrase it as "and BLS installation is explicitly disabled".

Reply:
I have renamed it.